src/core/src/service_interfaces/StatusHandler.py

```diff
@@ -25,14 +25,15 @@
 class StatusHandler(object):
     """Class for managing the core code's lifecycle within the extension wrapper"""
 
-    def __init__(self, env_layer, execution_config, composite_logger, telemetry_writer):
+    def __init__(self, env_layer, execution_config, composite_logger, telemetry_writer, vm_conext):
```

Comment:
There's a typo in context, but doesn't matter as consistency requires -- 
vm_cloud_type

(can discuss actual naming if you think there is something better)